src/app/static/src/app/utils.ts

```diff
@@ -191,6 +191,10 @@ export const formatDateTime = (isoUTCString: string) => {
     return moment.utc(isoUTCString).local().format('DD/MM/YYYY HH:mm:ss');
 };
 
+export const formatToISODateTime = (isoUTCString: string) => {
+    return moment.utc(isoUTCString).local().format('YYYY/MM/DD HH:mm:ss');
```

Comment:
So we want to convert our UTC ISO strings to local ISO strings for the note outputs? Do we definitely want ISO format? Just our standard format as shown in the app might be friendlier if these things are basically human-readable

Reply:
@r-ash has advocated for moving the standard format  as well. For now, rehydrate endpoint expects this format  `YYYY/MM/DD HH:mm:ss`, so will leave as is.

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3302